src/components/translation/TranslationService.js

```diff
@@ -21,8 +21,10 @@ goog.require('ga_topic_service');
 
       // Verify if a language is supported by the current topic
       var isLangSupportedByTopic = function(lang, topic) {
-        if (!topic) {
+        if (!topic && gaGlobalOptions.languages.indexOf(lang) > -1) {
```

Comment:
I think we need this test [here](https://github.com/ioda-net/mf-geoadmin3/blob/add-languages-globalVariables/src/components/translation/TranslationService.js#L18) instead  after we set the lang from the browser or permalink.